configs/cargo-deny/deny.toml

```diff
@@ -9,22 +9,14 @@ allow = [
     "BSL-1.0",
     "ISC",
     "MIT",
-    "OpenSSL",
     "Unlicense",
+    "Unicode-3.0",
```

Comment:
@cbgbt is this OK or should I use the large list of exceptions from the settings SDK?

Reply:
I just went ahead and made it the same since it was easy to copy/paste the exceptions list.